### PR TITLE
Poetry extension `README`

### PR DESCRIPTION
**Pull request by andreaphylum**

### CODE REVIEW

File: extensions/poetry/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## How it works
+
+When invoking `phylum poetry`, subcommands that would modify the
+`pyproject.toml` or `poetry.lock` files will trigger a Phylum analysis.
+
+- If the analysis is successful, the corresponding changes will be applied.
+- If the analysis is unsuccessful because some of the new dependencies don't
+  meet the required project thresholds, the command will exit with the error code
+  127.
```
Comment (cd-work) [inline extensions/poetry/README.md:36]: Not sure we should mention the error code explicitly here. I wouldn't want to nail myself down on that error code being 100% exactly this forever. Ultimately it shouldn't matter to users and if they need it in a script they can still check at their own risk.

File: extensions/poetry/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- If the analysis is waiting for Phylum to process one or more of the submitted
+  packages, the command will exit with the error code 126 and the changes will
+  _not_ be applied.
```
Comment (cd-work) [inline extensions/poetry/README.md:39]: Same exit code concerns here. I'd just use `the command will fail` as wording for these.

**Review by cd-work — CHANGES_REQUESTED** [review]
I think that should be mostly good to go tbh.

File: extensions/poetry/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Phylum Poetry extension
+
+A [Phylum CLI](phylum-cli) extension that checks your [Poetry][poetry]
```
Comment (maxrake) [inline extensions/poetry/README.md:3]:
```suggestion
A [Phylum CLI][phylum-cli] extension that checks your Python [Poetry][poetry]
```

File: extensions/poetry/README.md (hunk 1 above)
Comment (andreaphylum) [reply extensions/poetry/README.md]: Agree -- though at some point we may want to document that there _is_ indeed a difference between the two failures, if/when we do intend to explicitly expose that to users.

**Review by maxrake — CHANGES_REQUESTED** [review]
The wording gets the main points across and should be good for this first release. One suggested change in wording and one broken link reference to fix.

**Review by maxrake — APPROVED** [review]

**Review by cd-work — APPROVED** [review]

**Pull request merged**